dev-python/jupyter_core/jupyter_core-4.6.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Core common functionality of Jupyter projects"
+HOMEPAGE="https://jupyter.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
```

Comment:
You have dropped `arm64` from the previous version.  Could you please justify that?

Reply:
Sorry about that, 

I'm currently not at home, because of the Christmas holidays. I'll be back in about a week, I will fix this then.

Reply:
No worries. I take it as a mistake during your version bump.  I will push a fix.

Reply:
Awesome, thank you